client/extensions/woocommerce/app/product-categories/create.js

```diff
@@ -69,17 +77,42 @@ class ProductCategoryCreate extends React.Component {
 		}
 	}
 
-	onSave = () => {};
+	onSave = () => {
+		const { site, category, translate } = this.props;
+		this.setState( () => ( { busy: true } ) );
+
+		const successAction = () => {
+			page.redirect( getLink( '/store/products/categories/:site', site ) );
+			return successNotice( translate( 'Category successfully created.' ), {
```

Comment:
Hi! I've found a possible matching string that has already been translated **2** times:
`translate( 'Your post was successfully created.' )` *ES Score: 7*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/store-product-cat-save)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).